test/json-stream.js

```diff
@@ -0,0 +1,277 @@
+const t = require('tap')
+const { JSONStreamError, parse } = require('../lib/json-stream.js')
+
+t.test('JSONStream', (t) => {
+  t.test('JSONStreamError constructor', (t) => {
```

Comment:
This may be dead code.  Now that this isn't an external module it doesn't have to account for being used in any other way than we are using it here.